packages/astro/src/assets/fonts/utils.ts

```diff
@@ -132,8 +132,10 @@ export async function generateFallbacksCSS({
 	// eslint-disable-next-line @typescript-eslint/no-shadow
 	generateFontFace,
 }: {
-	/** The family name */
-	family: string;
+	family: {
+		name: string;
+		as?: string;
+	};
```

Comment:
Could we still allow the string version as shorthand?

Reply:
We could but I don't think it'd be more convenient here, because this function contains a call to `getFamilyName()` which expects this shape. In practise, using a shortand would only make sense in tests but we'd have to test the full shape anyways

Reply:
I put this comment in the wrong place. I meant the user config. Can they still use the shorthand version?

Reply:
I believe this is the same question I did in another comment. I believe the unit tests confused us 🤔

Reply:
Yes! A user can still provide `"Roboto"` and it will become `{ name: "Roboto" }`. Apologies for the confusion